gwpy/astro/range.py

```diff
@@ -21,22 +21,48 @@
 """
 
 import warnings
+
 from functools import wraps
 from math import pi
+from scipy.integrate import trapz
 
-from scipy import integrate
-
-from astropy import (units, constants)
+from astropy import (
+    units,
+    constants,
+)
```

Comment:
Please reinstate the blank line between the standard library imports and the `scipy` import.

Reply:
Done